internal/operators/cnv/manifest.go

```diff
@@ -57,6 +63,16 @@ func Manifests(config Config) (map[string][]byte, []byte, error) {
 
 	openshiftManifests := make(map[string][]byte)
 
+	if shouldInstallHPP(config, cluster) {
+		cnvHpp, err := hpp(config.SNOPoolSizeRequestHPPGib)
+		if err != nil {
+			return nil, nil, err
+		}
+		// Add HostPathProvisioner to CNV manifest
+		cnvHco = append(cnvHco, []byte("\n---\n")...)
+		cnvHco = append(cnvHco, cnvHpp...)
+		openshiftManifests["99_openshift-cnv_hpp_sc.yaml"] = []byte(cnvHPPStorageClass)
```

Comment:
Why 99?
@romfreiman  if i remember correctly we wanted to avoid using `99_`  prefix right?

Reply:
I think we decided this should be handled outside of this PR?
https://github.com/openshift/assisted-service/pull/3131#discussion_r777884854
https://github.com/openshift/assisted-service/pull/3131#discussion_r777424333

Reply:
OK do you have a ticket for this?

Reply:
https://issues.redhat.com/browse/MGMT-8851

Reply:
Thanks!